zstacklib/zstacklib/scripts/rbd_rw.py

```
@@ -0,0 +1,242 @@
+
+def write_rbd_image(pool_name, image_name, fs_id, offset=0, content=None):
+    image = get_image(pool_name, image_name, fs_id)
+    _do_write_image(image, offset, content)
+
+
+def _do_read_image(image, offset, size):
+    data_ptr = ctypes.create_string_buffer(size)
+    def oncomplete(_, data):
+        data_ptr.raw = data
+    completion = image.aio_read(offset, size, oncomplete)
+    for i in xrange(100):
```

Ruff (0.8.2)

Loop control variable `i` not used within loop body

Rename unused `i` to `_i`

(B007)